scripts/variantstore/tieout/GIAB_TIEOUT.md

```diff
@@ -0,0 +1,175 @@
+# NOTES
+ - Previously there was an attempt to tie out the feature extract input to VQSR between WARP and GVS.  However, that was abandoned in favor of the truth-sample based comparison here due to non-determinism and bugs in GenomicsDB that were hard to compensate for.  However, the docs and scripts for doing this are in the branch `gvs_vqsr_input_tieout` under `scripts/variantstore/tieout/feature_extract`
+ 
+## Prerequisites
+
+ - rtg
+ - bcftools
+ - tabix
+ - python 3.7+
+
```

Comment:
my immediate reaction was that I should pip install them, but that's clearly wrong.
I know I'm a noob with in this project, but a little more context on the prereqs would be immensely helfpul and would have saved me a lot of time googling

Reply:
add samtools?

Reply:
conda create --name gvs python=3.8
conda activate gvs
conda install -c bioconda samtools=1.9 --force-reinstall
conda install -c bioconda bcftools
conda install -c bioconda rtg-tools

Reply:
also add GATK to prereqs